theano/gpuarray/tests/tstgpueye.c

```diff
@@ -34,13 +34,10 @@ int APPLY_SPECIFIC(tstgpueye)(PyArrayObject *n, PyArrayObject *m,
   if (*z == NULL)
     return -1;
 
-  args[0] = (*z)->ga.data;
-  args[1] = &dims[0];
-  args[2] = &dims[1];
   ls = 1;
   gs = 256;
-  /* The k_eye name comes from the kernel declaration above. */
-  err = GpuKernel_call(&k_eye, 1, &ls, &gs, 0, args);
+  /* The eye_call name comes from the kernel declaration above. */
+  err = eye_call(1, &ls, &gs, 0, (*z)->ga.data, dims[0], dims[1]);
```

Comment:
1) want to take gpudata pointers to allow people to work with low-level gpudata object directly if necessary.  Other this will force them to use the GpuKernel_call interface for that.

2) ok.